config/package.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Lets us use the same MEANS of accessing config between browser and server `require('../config')` 
Though importantly we can **limit what gets included in the browser js bundle.** 
See the config/browser.js - that's all they will see.